xml/deployment_yast_installer.xml

```diff
@@ -2460,24 +2460,28 @@ sle-live-patching 8c541494</screen>
    <tip>
     <title>Checking policy compliance during installation</title>
     <para>
-     If you don not want to wait for the <guimenu>Installation
+     If you do not want to wait for the <guimenu>Installation
       Settings</guimenu> screen but want the installer to check the settings
      from the beginning of the installation process, boot the system with the
      boot parameter
      <literal>YAST_SECURITY_POLICY=<replaceable>POLICY</replaceable></literal>.
-     To check for compliance with the &disa; &stiga;, use 
+     To check for compliance with the &disaa; &stiga;, use 
      <literal>YAST_SECURITY_POLICY=stig</literal>. For more information about
      boot parameters, refer to <xref linkend="cha-boot-parameters"/>.
     </para>
    </tip>
    <para>
     The installer does not check all rules of the profile but only those
-    necesary for the installation or that are hard to fix afterward. To apply
+    necessary for the installation or that are hard to fix afterward. To apply
     the remaining rules, a full SCAP remediation is performed on first boot. You
     can also perform a <guimenu>scan only</guimenu> or <guimenu>do
      nothing</guimenu> and manually remediate the system later with &openscap;.
-    For more information, refer to the article
+    For more information, refer to the articles
```

Comment:
Why plural? It is only one article, isn't it?

Reply:
No, it's two articles, one about OpenSCAP and one about STIG.